arches/install/arches-templates/project_name/webpack/webpack-utils/build-filepath-lookup.js

```diff
@@ -0,0 +1,39 @@
+const Path = require('path');
+const fs = require('fs');
+
+function buildFilePathLookup(path, staticUrlPrefix) {
+    if (!fs.existsSync(path)) {
+        return;
+    }
+
+    let prefix = path.match(/[^\/]+$/)
```

Comment:
nit throughout -- default to `const` and use `let` only if expecting reassignment